ass/src/main/java/cz/ackee/ass/activity/FeedbackActivity.kt

```diff
@@ -152,15 +160,24 @@ internal class FeedbackActivity : AppCompatActivity() {
     }
 
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
-        if (requestCode == RC_SCREENSHOT_EDIT) {
-            if (resultCode == Activity.RESULT_OK) {
-                data?.getParcelableExtra<Uri>(EditActivity.SCREENSHOT_BITMAP_URI)?.let {
-                    Picasso.get().invalidate(it)
-                    Picasso.get().load(it).into(imgScreenshot)
+        when (requestCode) {
+            RC_SCREENSHOT_EDIT -> {
+                if (resultCode == Activity.RESULT_OK) {
+                    data?.getParcelableExtra<Uri>(EditActivity.SCREENSHOT_BITMAP_URI)?.let { uri ->
+                        intent.putExtra(ARG_FEEDBACK_DATA, feedbackData.copy(screenshotUri = uri))
```

Comment:
are you sure this will survive orientation change/process restoration? I'm not sure if this Intent is mutable

Reply:
Tested it